Makefile

```diff
@@ -1,9 +1,17 @@
-QUAY_REPO=$(USER)
-IMAGE_NAME=jupyterhub-img
-IMAGE_TAG=test-jsp
+ifneq ("$(wildcard ./.env.local)","")
+	include ./.env.local
+endif
+
+QUAY_REPO ?= $(USER)
+GIT_USER ?= $(USER)
 NAMESPACE ?= $(USER)-odh
+OPERATOR_NAME ?= odh-operator
+OPERATOR_NAMESPACE ?= odh-ods-operator
 GIT_REF ?= master
-GIT_USER ?= $(USER)
+REMOTE_CMD ?= podman
```

Comment:
Why is it called `REMOTE_CMD` when it's used in local build?

Reply:
Good question, I'll update to `LOCAL_CMD`